ensime_shared/util.py

```diff
@@ -56,3 +57,17 @@ def module_exists(module_name):
         __import__(module_name)
         res = True
     return res
+
+
+class Pretty(object):
+    """Wrapper to pretty-format object's string representation.
+
+    Reduces boilerplate for logging statements where we don't want to eagerly
+    :func:`pprint.pformat` when the logging level isn't enabled.
+    """
+
+    def __init__(self, data):
+        self._data = data
+
+    def __str__(self):
```

Comment:
Doesn't seem to work, that's probably happening within `pprint` for understandable reasons of accuracy. It should go away in Python 3… 😁